src/constants/tokens.ts

```diff
@@ -374,6 +437,19 @@ function getCeloNativeCurrency(chainId: number) {
   }
 }
 
+export function isBoba(chainId: number): chainId is SupportedChainId.BOBA {
+  return chainId === SupportedChainId.BOBA
+}
+
+function getBobaNativeCurrency(chainId: number) {
```

Comment:
Boba's default native currency is $ETH, and $BOBA is the alt fee token
Is there any fee switching logic involved/required?